tests/AppBundle/Utils/JobeetTest.php

```diff
@@ -3,19 +3,36 @@
 namespace Tests\AppBundle\Utils;
 
 use AppBundle\Utils\Jobeet;
+use Doctrine\Common\Annotations\Annotation\IgnoreAnnotation;
 use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
 
+/**
+ * @IgnoreAnnotation("dataProvider")
```

Comment:
why do you need `IgnoreAnnotation` here?

Reply:
I read about caveats with annotations here https://github.com/liip/LiipFunctionalTestBundle/tree/2.x#caveats
But I removed these annotations and tests still pass.